dotcom-rendering/src/web/components/Callout/CalloutDescription.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import { textSans, space, body } from '@guardian/source-foundations';
+import { decidePalette } from '../../lib/decidePalette';
+
+const descriptionStyles = (format: ArticleFormat) =>
+	css`
+		a {
+			color: ${decidePalette(format).text.richLink};
+			border-bottom: 1px solid ${decidePalette(format).text.richLink};
+			text-decoration: none;
+		}
+		${textSans.small()}
+		padding-bottom: ${space[4]}px;
+		${body.medium()}
```

Comment:
Do we need this with the textSans on line 12?

Reply:
we do not - thanks